debian/changelog

```diff
@@ -1,3 +1,103 @@
+snapcraft (2.0) xenial; urgency=medium
+
+  * Updating debian/changelog for the 2.0 release
+
+ -- Leo Arias <yo@elopio.net>  Mon, 18 Jan 2016 11:59:28 -0600
+
+snapcraft (2.0) xenial; urgency=medium
+
+  [ Daniel Holbach ]
+  * Add document detailing the snapcraft.yaml syntax.
+  * Fix typo - thanks Kyle.
+  * Add missing periods.
+  * Add missing 'to'.
+  * Rephrase sentence based on Leo's feedback.
+  * Remove vendor for 'master'.
+  * Add document about parts.
+  * Add link to config docs.
```

Comment:
Nahh, I wouldn't say to rephrase anything (so they can be matched up with commits), but things like "Fix typo - thanks Kyle." That's not helpful-- it's changelog noise. I'd remove things like that.